mapswipe_workers/tests/integration/test_create_build_area_project.py

```diff
@@ -18,22 +18,29 @@ def setUp(self):
     def tearDown(self):
         tear_down.delete_test_data(self.project_id)
 
-    def test_create_project(self):
+    def test_create_build_area_project(self):
         runner = CliRunner()
         runner.invoke(mapswipe_workers.run_create_projects)
 
         pg_db = auth.postgresDB()
-        query = "SELECT project_id FROM projects WHERE project_id = '{0}'".format(
-            self.project_id
-        )
-        result = pg_db.retr_query(query)[0][0]
+        query = "SELECT project_id FROM projects WHERE project_id = %s"
```

Comment:
Do you need single quotes around `%s` in the query? Or the pg driver adds them automatically?

Reply:
Hey,
no this is not needed here. psycopg2 will take care of this.